src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -39,22 +39,384 @@ public with sharing class AFFL_MultiRecordType_TEST {
     
     private static ID orgRecTypeID;
     private static ID householdRecTypeID;
-    
-    public static void setup() {
-        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getHhAccRecTypeID()));
-                
-        List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
-        mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Business Organization', Primary_Affl_Field__c = 'Primary Business Organization'));	
-        mappings.add(new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'Household Account', Primary_Affl_Field__c = 'Primary Household'));
-        insert mappings;
 
-        orgRecTypeID = UTIL_Describe_API.getBizAccRecTypeID();
-        householdRecTypeID = UTIL_Describe_API.getHhAccRecTypeID();
+/**************************************************************************
+* SETUP
+**************************************************************************/
+
+    /**************************************************************************
+    * @description Sets up hierarchy settings and affiliation mappings for
+    * tests using record type name and contact field api name
+    * @return The affiliation mapping wrappers set up for the tests.
+    **************************************************************************/
+    public static List<AffiliationMappingWrapper> setupAndGetMappingsByDevNameAndApiName() {
+        AFFL_MultiRecordType_TEST.initializeAccountRecordTypeIds();
+
+        List<AffiliationMappingWrapper> affiliationMappingWrapperList =
+            AFFL_MultiRecordType_TEST.getAfflMappingWrapperByDevNameAndApiName();
+
+        return AFFL_MultiRecordType_TEST.setupAndGetMappings(affiliationMappingWrapperList);
```

Comment:
Also, I am not sure why we want to setupAndGetMappings in setupAndGetMapping...Maybe we want to give a better name to setupAndGetMappings()

Reply:
Renamed this to commonSetup to be much more explicit that it is a set of shared setup steps. We can probably rename this further down the line to something even better.